README.md

```diff
@@ -85,3 +85,10 @@ var intValue : Int
 
 ```
 
+Colons should be centered when used in other cases, such as dictionary literals:
+
+```swift
+
+var myDictionary: [String : AnyObject] = ["Hello" : Object()]
```

Comment:
This would be inconsistent with the the other uses of `:`. Eg. `func foo(sender: UIButton)`, `var fooVar: UIButton`, etc.  I lean towards maintaining the consistency.